CHANGELOG.md

```diff
@@ -1,12 +1,24 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
Lots of unrelated changes, but no `typescript` ? :stuck_out_tongue_winking_eye:

Reply:
lol no, it prepares for typescript: the only thing to do now is to rename file from js to ts.

Almost all the changes (removing `@lingui/macro`) is for typescript compat.

And well typescript stuff is not much useful in changelog imo